### DISCUSSION TITLE
Missing fields : BT-770, BT-715, BT-725 ,BT-716 (from (EU) 2019/1780)

### DISCUSSION THREAD

**Discussion by Tronquill** [OP-TED/eForms-SDK#774] [Q&A] [answered]
Hi, As part of the national customization (tailoring), we need to mark the BT-770 (Organization Subrole), BT-715 (Vehicles), BT-725 (Vehicles Zero Emission) and BT-716 (Vehicles Clean) fields as required (https://docs.ted.europa.eu/eforms/latest/reference/business-terms/index.html). I could not find these fields in the definition of all fields in the SDK. Do you plan to add them to the SDK in the near future, and if not, did you plan a way to expand the definition of fields and notification type fields in the SDK for developers (we don't want to break cooperation during version updates)?

**Comment by YvesJo** [comment, marked as answer]
Hi,
BT-770, BT-715 .... are in fact Business Terms.
Most of he fields associated to the roles and subroles have an ID of the form "OPT-301-...", some are identified using indicators and very few (like winner) involve a combination of multiple fields; this approach also allows to provide the required context (as roles/subroles of an organization may also vary per lot) and results of a tradeoff between the Implementing Regulation annex and the UBL Standard requirements. You may find information on roles and subroles in the documentation at [roles](https://docs.ted.europa.eu/eforms/latest/schema/parties.html#_roles) and [subroles](https://docs.ted.europa.eu/eforms/latest/schema/parties.html#_sub_roles) respectively
BT-715, BT-716 and BT-725 are addressed using the 2 fields OPT-155-LotResult and OPT-156-LotResult as described in the [LotResult](https://docs.ted.europa.eu/eforms/latest/schema/competition-results.html#_lot_result) section.
Given the constraints given by the Implementing Regulation and the Standard, there is no plan to change this.
KR,

**Reply by Tronquill** [reply]
thank you for the full explanation ! :)
